sparta/sparta/resources/CircularBuffer.hpp

```diff
@@ -265,6 +265,18 @@ namespace sparta
                 sparta_assert(isValid(), "Iterator is not valid for dereferencing");
                 return circularbuffer_entry_->data;
             }
+            value_type* operator->() {
```

Comment:
opening brace on the next line to match with existing style

Reply:
Fixed in 2295bdccdb872